actions/diaspora

```diff
@@ -20,11 +20,14 @@
 Configuration helper for diaspora* pod.
 """
 
+import os
 import argparse
 import subprocess
 
 from plinth import action_utils
 
+GEMFILE = '/usr/share/diaspora/Gemfile'
```

Comment:
I don't think we should modify any files in /usr, except during installation.

Why is this change needed? Is it a bug in the diaspora package?